qualtran/bloqs/phase_estimation/lp_resource_state_test.py

```diff
@@ -42,99 +41,64 @@ def test_lp_resource_state_auto(bloq_autotester):
 
 def test_lp_resource_state_symb():
     bloq = _lp_resource_state_symbolic.make()
-    assert bloq.t_complexity().t == 4 * bloq.bitsize
+    assert bloq.t_complexity().t == 4 * bloq.bitsize + 4
```

Comment:
why + 4?

Reply:
n is considered to be `n+1` because I now use `ReflectionUsingPrepare` on instead of directly synthesizing the reflection (which was happening earlier). The delegation to `ReflectionUsingPrepare` is slightly inefficient, but it's cleaner and we can improve this efficiency later